c_src/quicer_ctx.c

```diff
@@ -257,7 +257,7 @@ destroy_s_ctx(QuicerStreamCTX *s_ctx)
   enif_free_env(s_ctx->imm_env);
   // Since enif_release_resource is async call,
   // we should demon the owner now!
-  enif_demonitor_process(s_ctx->env, s_ctx, &s_ctx->owner_mon);
+  // enif_demonitor_process(s_ctx->env, s_ctx, &s_ctx->owner_mon);
```

Comment:
We could remove it because we set stream handle to NULL before destroy and also
A monitor is automatically removed when it triggers or when the resource is deallocated.